velox/docs/functions/spark/datetime.rst

```diff
@@ -213,6 +213,35 @@ These functions support TIMESTAMP and DATE input types.
 
         SELECT month('2009-07-30'); -- 7
 
+.. spark:function:: months_between(timestamp1, timestamp2) -> double
+
+    	If `timestamp1` is later than `timestamp2`, then the result is positive.
+    	If `timestamp1` and `timestamp2` are on the same day of month, or both are the last day of month, time of day will be ignored.
+    	Otherwise, the difference is calculated based on 31 days per month, and rounded to 8 digits ::
+
+        SELECT months_between("1970-05-01 00:00:01", "1970-06-01 12:00:01"); -- -1
+        SELECT months_between("1970-07-01 00:00:01", "1970-06-01 12:00:01"); -- 1
+
+.. spark:function:: months_between(timestamp1, timestamp2, roundOff) -> double
```

Comment:
I assume 'roundOff' is must-have parameter, as Spark provides default true if the user does not specify it. I suppose we can combine the three documentations to avoid duplicate, and the signature would be like `months_between(timestamp1, timestamp2, roundOff[, timezone])` where timezone is optional.

Reply:
Done